runtime/src/bank.rs

```diff
@@ -7691,6 +7691,46 @@ impl Bank {
             .epoch_accounts_hash_manager
             .try_get_epoch_accounts_hash()
     }
+
+    /// Write details about this Bank's stored accounts to a file
+    pub fn write_stored_accounts_to_file_for_debugging(&self) {
+        assert!(
+            self.is_frozen(),
+            "Bank must be frozen to print stored accounts"
+        );
+        let slot = self.slot();
+
+        let file_name = format!("{}.{}", slot, self.hash());
+        let path = self
+            .rc
+            .accounts
+            .accounts_db
+            .get_accounts_hash_cache_path()
+            .join(&file_name);
```

Comment:
I was going to rename this directory in another PR, but think we should drop another directory here? Make it something like `/ledger/debug` ?

Reply:
maybe /ledger/bank_hash_debug